Sources/Markdown/Block Nodes/Block Container Blocks/Doxygen Commands/DoxygenParam.swift

```diff
@@ -0,0 +1,78 @@
+/*
+ This source file is part of the Swift.org open source project
+
+ Copyright (c) 2023 Apple Inc. and the Swift project authors
+ Licensed under Apache License v2.0 with Runtime Library Exception
+
+ See https://swift.org/LICENSE.txt for license information
+ See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+*/
+
+import Foundation
+
+/// A parsed Doxygen `\param` command.
+///
+/// The Doxygen support in Swift-Markdown parses `\param` commands of the form
+/// `\param name description`, where `description` extends until the next blank line or the next
+/// parsed command. For example, the following input will return two `DoxygenParam` instances:
+///
+/// ```markdown
+/// \param coordinate The coordinate used to center the transformation.
+/// \param matrix The transformation matrix that describes the transformation.
+/// For more information about transformation matrices, refer to the Transformation
+/// documentation.
+/// ```
+public struct DoxygenParam: BlockContainer {
```

Comment:
nit: Spelling this out as `DoxygenParameter` might be nicer. `param` is the lexical token that Doxygen uses, but we can refer to it as a "parameter" for clarity / consistency with Swift API naming guidelines.